### PR TITLE
Add support for atmospheric polarization

### PR DESCRIPTION
**Pull request by keskitalo**
This PR adds `polarization_fraction` as a trait to the atmospheric simulation. It is a constant fraction that generates Q-polarization from the pure intensity simulation.

### CODE REVIEW

File: workflows/toast_sim_ground.py [hunk 1]
```diff
@@ -309,6 +314,9 @@ def main():
             out_model="el_noise_model",
         ),
         toast.ops.PointingDetectorSimple(name="det_pointing_azel", quats="quats_azel"),
+        # In the future, `det_weights_azel` may be a dedicated operator that does not
+        # expand pixel numbers but just Stokes weights
+        toast.ops.PointingHealpix(name="det_weights_azel", mode="IQU"),
```
Comment (tskisner) [inline workflows/toast_sim_ground.py:319]: Note that unless the user specifies `--det_weights_azel.weights some_other_string`, the default name of the detector weights computed by this operator (`weights`) is the same as that used by the `pointing` operator below.  Perhaps we could set `weights="weights_azel"` or similar in the constructor?

**Review by tskisner — APPROVED** [review]
Just one small comment.  Also, should we add this feature to the ground benchmark?

File: workflows/toast_sim_ground.py (hunk 1 above)
Comment (keskitalo) [reply workflows/toast_sim_ground.py]: Done.

**Comment by keskitalo** [comment]
I would hold off on adding polarized atmosphere to the benchmarks. It is likely to evolve quite a bit.

**Pull request merged**